### PR TITLE
fix CRAN issues

### PR DESCRIPTION
**Pull request by sbfnk**
This picks out the changes in #308 relating to fixing of CRAN issues, and eliminates all remaining WARNINGS. This now passes all checks on winbuilder and rhub.

I'd suggest to submit this as version 1.3.3 now and integrate the new features in #308 (and/or #305) towards a later 1.3.4 release.

### CODE REVIEW

File: tests/testthat/test-stan-rt.R [hunk 1]
```diff
@@ -1,61 +1,59 @@
 skip_on_cran()
-suppressMessages(expose_stan_fns("rt.stan", target_dir = system.file("stan/functions", package = "EpiNow2")))
-
 
 # Test update_Rt
 test_that("update_Rt works to produce multiple Rt estimates with a static gaussian process", {
   expect_equal(
-    update_Rt(rep(1, 10), log(1.2), rep(0, 9), rep(10, 0), numeric(0), 0),
```
Comment (seabbs) [inline tests/testthat/test-stan-rt.R:8]: There are no Stan level changes in this PR. I don't know where the change to update_Rt is now so perhaps already on main.

**Review by seabbs — APPROVED** [review]
This looks good in the main. There are a few issues that I think may slip through the cran check but still need fixing but think probably best to merge and fix as a follow up.

File: R/estimate_secondary.R [hunk 2]
```diff
@@ -53,30 +53,12 @@
...
 #' # make some example secondary incidence data
 #' cases <- example_confirmed
-#' cases <- as.data.table(cases)
+#' cases <- as.data.table(cases)[, primary := confirm]
+#' # Assume that only 40 percent of cases are reported
+#' cases[, scaling := 0.4]
+#' # Parameters of the assumed log normal delay distribution
+#' cases[, meanlog := 1.8][, sdlog := 0.5]
 #'
-#' # apply a convolution of a log normal to a vector of observations
```
[the remark this answers is not in the document]
Comment (sbfnk) [reply R/estimate_secondary.R:58]: Not sure I understand but `example(estimate_secondary)` is working fine on `master`. As you've probably seen the simulation function is in https://github.com/epiforecasts/EpiNow2/pull/310 with the next release as target.

File: tests/testthat/test-stan-rt.R (hunk 1 above)
Comment (sbfnk) [reply tests/testthat/test-stan-rt.R:8]: Yes, this was in #303 but the tests weren't updated then.

File: R/estimate_secondary.R (hunk 2 above)
[the remark this answers is not in the document]
Comment (seabbs) [reply R/estimate_secondary.R:58]: Hmm okay, I must have misunderstood this then.

[the remark this answers is not in the document]
Comment (sbfnk) [reply R/estimate_secondary.R:58]: I think you hadn't - checking again the example doesn't work after all. This is now addressed in https://github.com/epiforecasts/EpiNow2/commit/41dd4e8e80435e95dc9642828bd345c6fad2d313.

**Pull request merged**